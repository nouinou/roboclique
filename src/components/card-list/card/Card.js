const Card = (props) => {
	const { user } = props;

	return (
		<div className="flex flex-col justify-center items-center w-100 bg-slate-100 m-0.5 p-10 cursor-pointer hover:animate-wiggle rounded-xl">
			<img
				className="robo-img text-center w-avatar"
				src={user.image}
				alt="robot"
			/>
			<span className="robo-text text-primary text-2xl font-bold capitalize">
				{user.fullname}
			</span>
			<a href={`mailto:${user.email}`} className="text-md hover:text-primary">
				{user.email}
			</a>
		</div>
	);
};

export default Card;
