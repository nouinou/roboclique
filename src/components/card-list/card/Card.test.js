import { shallow } from 'enzyme';
import Card from './Card';

describe('Card Component', () => {
	const mockRobot = {
		fullname: 'Miaw Bot',
	};
	it('should render Card', () => {
		expect(shallow(<Card robot={mockRobot} />)).toMatchSnapshot();
	});
});
