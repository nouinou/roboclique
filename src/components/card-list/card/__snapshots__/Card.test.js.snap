// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component should render Card 1`] = `
<div
  className="flex flex-col justify-center items-center w-100 bg-slate-100 m-0.5 p-10 cursor-pointer hover:animate-wiggle rounded-xl"
>
  <img
    alt="robot"
    className="robo-img text-center w-avatar"
    height="192"
    width="192"
  />
  <span
    className="robo-text text-primary text-2xl font-bold capitalize"
  >
    Miaw Bot
  </span>
  <a
    className="text-md hover:text-primary"
    href="mailto:undefined"
  />
</div>
`;
